zcutil/fetch-params.sh

```diff
@@ -107,13 +107,27 @@ function fetch_params {
 
     if ! [ -f "$output" ]
     then
-        for method in wget ipfs curl failure; do
-            if "fetch_$method" "$filename" "$dlname"; then
-                echo "Download successful!"
-                break
+        for i in 1 2
+        do
+            for method in wget ipfs curl failure; do
+                if "fetch_$method" "${filename}.part.${i}" "${dlname}.part.${i}"; then
+                    echo "Download of part ${i} successful!"
+                    break
+                fi
+            done
+        done
+
+        for i in 1 2
+        do
+            if ! [ -f "${dlname}.part.${i}" ]
+            then
+                fetch_failure
             fi
         done
 
+        cat "${dlname}.part.1" "${dlname}.part.2" > "${dlname}"
```

Comment:
(Maybe) instead move `${dlname}.part.1` to `${dlname}` and then append `${dlname}.part.2` to it, then delete `${dlname}.part.2`. This maybe helps reduce disk usage.

Reply:
Disks are cheap.

Reply:
We can always improve this later.